src/index.js

```diff
@@ -1,10 +1,46 @@
-import _ from 'lodash';
-import './style.css';
+import './style.scss';
 
-function component() {
-    const element = document.createElement('div');
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    return element;
+const list = [{
+  desc: 'This is the area where you should name your Task',
+  comp: false,
+  index: 0,
+},
+{
+  desc: 'Task 2',
+  comp: false,
+  index: 0,
+},
+{
+  desc: 'Task 3',
+  comp: true,
+  index: 0,
+}];
+
+function displayTasks() {
```

Comment:
- [ ] The array of objects is here but the codes are not taking it.

![kk](https://user-images.githubusercontent.com/61562626/144463132-71d5d212-4f96-4716-befc-36b57dbf6b67.jpeg)

Reply:
Thanks for the reviwe :+1: 
somehow when i installed the sass-loader package it didn't saved into `package.json` file.

now its fixed :ok: